recipes-kernel/linux/linux-raspberrypi_6.6.bbappend

```diff
@@ -0,0 +1,18 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+COMPATIBLE_MACHINE:raspberrypi5 = "(raspberrypi5)"
+
+DOMU_DTB_NAME = "${RPI_SOC_FAMILY}-${MACHINE}-domu"
```

Comment:
what is domu-dtb? Are we going to have domu?

Reply:
It is DOMD_DTB_NAME in moulin

Reply:
Now it is DOMD

Reply:
Summary replace DOMU0 with DOMD

Reply:
Replaced

Reply:
not all. I've added comment.